src/NodeNameResolver/NodeNameResolver.php

```diff
@@ -71,11 +73,7 @@ public function isNames(Node $node, array $names): bool
      */
     public function isName(Node | array $node, string $name): bool
     {
-        if ($node instanceof MethodCall) {
-            return false;
-        }
-
-        if ($node instanceof StaticCall) {
+        if ($node instanceof CallLike) {
```

Comment:
`FuncCall` should be allowed to get name

Reply:
actually, the check can be moved to `isSingleName()` as the node can be array, so only single location for check

Reply:
good catch.